libmscore/score.cpp

```diff
@@ -4501,7 +4501,7 @@ void MasterScore::updateExpressive(MasterSynthesizer* m)
       SynthesizerState s = synthesizerState();
       SynthesizerGroup g = s.group("master");
 
-      int method;
+      int method = 0;
```

Comment:
@Jojo-Schmitz - could you change this? It should really be defaulting to 1, it was a mistake on my part to make it default to nothing (i.e. 0).

Reply:
done

Reply:
Thank you!